Assets/Scripts/Menu/MenuBird.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MenuBird : MonoBehaviour
+{
+    public Vector2 velocity = new Vector2(2.0f, 0);
+
+    private float screenEnds = 20;
```

Comment:
Would it be possible to derive this number via the camera view somehow to make it more dynamic and tied to the actual viewport?

Reply:
I have updated the script to derive the off screen positions of the camera